src/Database/CompanyName.MyMeetings.Database/meetings/Tables/MeetingCommentingConfigurations.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE meetings.MeetingCommentingConfigurations
+(
+    [Id]                    UNIQUEIDENTIFIER NOT NULL,
+    [MeetingId]             UNIQUEIDENTIFIER NOT NULL,
+    [IsCommentingEnabled]   BIT              NOT NULL,
+    CONSTRAINT [PK_meetings_MeetingCommentingConfigurations_Id] PRIMARY KEY ([Id] ASC),
+    CONSTRAINT [FK_meetings_MeetingCommentingConfigurations_Meetings] FOREIGN KEY ([MeetingId]) REFERENCES meetings.Meetings ([Id]) 
+        ON DELETE CASCADE 
+        ON UPDATE CASCADE
```

Comment:
Cascade deletes/updates are not good solutions in my opinion. 

1. The consistency and integrity should be provided and implemented on the **application** level - not database level. 
2. Cascading deletes/updates are **implicit** ("magic"). I prefer solutions when you need to be **explicit**. In that case, it is way easier to reason about the whole system.

Reply:
Ok, changed behavior here and in EF configuration for MeetingCommentingConfiguration.

Reply:
Andrei, you changed it in the migration script but you forgot about this file :)

Reply:
:) fixed